docs/defining-blocks.md

```diff
@@ -379,6 +386,39 @@ have side effects. This happens automatically when there is no initializer,
 or the initializer is a simple constant, but for function calls and constructors
 you have to include `whenUsed`.
 
+### Properties
+
+Fields and get/set accessors of classes defined in TypeScript can be exposed in blocks.
+Typically, you want a single block for all getters for a given type, a single block
+for setters, and possibly a single block for updates (compiling to the ``+=`` operator).
+This can be done automatically with `//% blockCombine` annotation, for example:
+
+```typescript
+class Foo {
+    //% blockCombine
+    x: number;
+    //% blockCombine
+    y: number;
+    // exposed with custom name
+    //% blockCombine="foo bar"
```

Comment:
I think the localization extraction logic only looks for "block" field so we should be using that instead.

Reply:
`block=...` will (or should) expose it as a regular block; I guess we could have `block="..." combine`?

Reply:
Yes that would work. It’ll get localized automatically.

Reply:
hmm, then I guess we need `//% block combine`? or maybe just `//% combine` ?

Reply:
Either should work but we need that stuff to be localizable.